app/uk/gov/hmrc/helptosavefrontend/models/eligibility/EligibilityCheckResult.scala

```diff
@@ -18,35 +18,18 @@ package uk.gov.hmrc.helptosavefrontend.models.eligibility
 
 import play.api.libs.json.{Format, Json}
 
-sealed trait EligibilityCheckResult {
-  val value: EligibilityCheckResponse
-}
+case class EligibilityCheckResult(result: String, resultCode: Int, reason: String, reasonCode: Int)
 
 object EligibilityCheckResult {
 
-  case class Eligible(value: EligibilityCheckResponse) extends EligibilityCheckResult
-
-  case class Ineligible(value: EligibilityCheckResponse) extends EligibilityCheckResult
+  implicit val format: Format[EligibilityCheckResult] = Json.format[EligibilityCheckResult]
 
-  case class AlreadyHasAccount(value: EligibilityCheckResponse) extends EligibilityCheckResult
+}
 
-  object Eligible {
-    implicit val format: Format[Eligible] = Json.format[Eligible]
-  }
+case class EligibilityCheckResponseAndThreshold(eligibilityCheckResult: EligibilityCheckResult, threshold: Option[Double])
```

Comment:
in the backend logic, this class is `EligibilityCheckResponse` , better to keep the same naming here to keep consistency

Reply:
ok I've changed it to that, thanks